scripts/monitoring/run_monitor_containers.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 path=$PWD
+webhook_url=''
 if [ ! -z "$1" ]
   then
     path=$1
 fi
 
+if [ ! -z "$2" ]
+  then
+    webhook_url=$2
+fi
```

Comment:
Can we please throw an error saying "Please enter webhook URL" if it is not passed.

Reply:
The python script throws an exception. Do we need additional logs?

Reply:
I see..then it is fine. I thought that it might not be handled.